### PR TITLE
Gui tests

### LINKED ISSUE #51: Add Gui Tests for floating tasks, undo/redo

### PR DESCRIPTION
**Pull request by louislai**
Resolve #51 

In progress

### CODE REVIEW

File: src/main/java/seedu/toluist/dispatcher/CommandDispatcher.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
     public void dispatch(Ui renderer, String command) {
         final Controller controller = getBestFitController(renderer, command);
         final CommandResult feedbackToUser = controller.execute(command);
+        System.out.print("dispatch " + command + " with result " + feedbackToUser);
```
Comment (Melvin-Tan) [inline src/main/java/seedu/toluist/dispatcher/CommandDispatcher.java]: i presume this is for logging purposes? is this the correct way to log? o.o

File: src/main/java/seedu/toluist/ui/MainWindow.java [hunk 2]
```diff
@@ -136,11 +129,17 @@ private void setWindowMinSize() {
...
-//        HelpWindow helpWindow = new HelpWindow();
-//        helpWindow.show();
-//    }
+    private void configureChildrenViews() {
+        taskListUiView = new TaskListUiView();
+        taskListUiView.setParent(getTaskListPlaceholder());
+
+        commandBox = new CommandBox(dispatcher);
+        commandBox.setParent(getCommandBoxPlaceholder());
+        commandBox.render();
```
Comment (Melvin-Tan) [inline src/main/java/seedu/toluist/ui/MainWindow.java]: Is something wrong with this? 
configureChildrenViews shouldnt concern itself with render right?

File: src/test/java/guitests/AddTaskCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import seedu.toluist.model.Task;
+
+/**
+ * Gui tests for add task command
+ */
+public class AddTaskCommandTest extends GuiTest {
+    @Test
+    public void addFloatingTask() {
+        String taskDescription = "do homework for Melvin";
```
Comment (Melvin-Tan) [inline src/test/java/guitests/AddTaskCommandTest.java]: Thanks friend

**Review by Melvin-Tan — COMMENTED** [review]
Havent review test portion, will come back to this

**Comment by louislai** [comment]
@Melvin-Tan eh this is in progress. I didnt add u as reviewer

**Comment by Melvin-Tan** [comment]
Orh PM me when it's ready to review then.

**Comment by louislai** [comment]
@Melvin-Tan ready for review

**Review by Melvin-Tan — APPROVED** [review]
Eyeballed finished, addressed most of my concerns on telegram already :)

**Pull request merged**